### PR TITLE
Merge start page into main

### PR DESCRIPTION
**Pull request by jbeccherini**
Added functionality to select your rocket and track (only canyon atm) and puts the rocket in the scene

### CODE REVIEW

File: Assets/Scripts/UI/StartMenu/RaceSelector.cs
Comment (Aether700) [inline Assets/Scripts/UI/StartMenu/RaceSelector.cs]: please avoid public fields. Here you could easily make a getter for this variable instead of making it public

**Review by Aether700 — CHANGES_REQUESTED** [review]
left a comment for some of the code, the UI scaling also needs some work:
image

**Review by Aether700 — COMMENTED** [review]
Good job. Just waiting on Jason to fix the scaling issue before I approve

**Comment by afourneaux** [comment]
- Rather than setting the canvas to Constant Size, may we have it scale with screen size? Just punch in the resolution you used to design the UI and it should scale well for everyone else.
- The camera on the Main Menu scene should be of a flat colour (ideally either black or the same red as the menu) rather than the default Unity skybox

**Comment by afourneaux** [comment]
Hovering over "Start Race" causes the canyon map to be highlighted

**Pull request closed without merging**
